### PR TITLE
Implement database support

### LINKED ISSUE #4: Implement database support
We need to implement some sort of database to our project in order to store information about games, players etc.

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR closes #4 

### What's done:

- [x] Added game.py that implements Game class
- [x] Added player.py that implements Player class
- [x] Added registration.py that implements Registration class
- [x] Added database.py that implements basic interactions with sqlite
- [x] Added tests

### CODE REVIEW

File: src/classes/game.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    date = datetime.datetime
+    place = str
+    max_players = int
+    description = str
+
+    def __init__(self, date: int or datetime.datetime, place: str, id: int = 0, max_players: int = 8, description: str = "") -> None:
+        """
+        Initialize Game object.
+
+        :param date: when game takes place
```
Comment (dgreflex) [inline src/classes/game.py:18]: Incorrect comment message

File: src/classes/game.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+"""Module that implements Game class."""
+import datetime
+
+
+class Game:
+    """Class that implements game."""
+
+    id = int
```
Comment (dgreflex) [inline src/classes/game.py]: Are you sure is should be like this `id = int` instead of this `id: int`

Comment (sanyavertolet) [reply src/classes/game.py]: Not sure, ok, will add annotations

File: src/classes/player.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+
+class Player:
+    """Class that implements player."""
+
+    id = int
+    name = str
+    is_adm = bool
+    games = int
+    pitch = float
```
Comment (dgreflex) [inline src/classes/player.py]: I believe this is a bug, instead of annotations you get objects which are classes.

**Review by dgreflex — CHANGES_REQUESTED** [review]
Fix errors in annotations in classes:
- Player in player.py
- Game in game.py
- Registration in registration.py

File: src/classes/player.py (hunk 3 above)
Comment (sanyavertolet) [reply src/classes/player.py]: Done

File: src/classes/game.py (hunk 2 above)
Comment (sanyavertolet) [reply src/classes/game.py]: Done

**Review by dgreflex — APPROVED** [review]
Nice job!

**Pull request merged**